src/psyclone/dynamo0p3.py

```diff
@@ -6193,31 +7273,17 @@ def field_bcs_kernel(self, function_space):
                 "Expected a gh_field from which to look-up boundary dofs "
                 "for kernel {0} but got {1}".format(self._kern.name,
                                                     farg.type))
-        new_parent, position = parent.start_parent_loop()
-        new_parent.add(AssignGen(new_parent, pointer=True,
-                                 lhs="boundary_dofs",
-                                 rhs=farg.proxy_name +
-                                 "%vspace%get_boundary_dofs()"),
-                       position=["before", position])
+        self._arglist.append("boundary_dofs_"+farg.name)
 
     def operator_bcs_kernel(self, function_space):
-        ''' Supply necessary additional arguments for the kernel that
-        applies boundary conditions to a LMA operator '''
-        from psyclone.f2pygen import DeclGen, AssignGen
+        '''
+        Supply necessary additional arguments for the kernel that
+        applies boundary conditions to a LMA operator.
```

Comment:
interface in docstring please.

Reply:
The only argument was in fact unused so I've renamed it to `_` and commented in the docstring.